packages/mdc-select/mdc-select.scss

```diff
@@ -114,14 +109,15 @@
     background-position: left 10px center;
   }
 
-  .mdc-select__surface {
+  .mdc-select__surface,
```

Comment:
Since this element is no longer the ripple surface, we might want to consider renaming it... e.g. checkbox has `__native-control` for this.

(We'd want to change variable names in JS too)